mysql/errcode.go

```diff
@@ -941,6 +943,4 @@ const (
 	ErrRegionUnavailable  = 9005
 	ErrGCTooEarly         = 9006
 	ErrWriteConflict      = 9007
-
-	ErrTxnTooLarge = 9500
```

Comment:
Why change this?

Reply:
It is not a related change. But ErrTxnTooLarge is a TiDB error, it should not in the TiKV/PD section.

Reply:
It is better to keep the TiDB errors starts from 8000 and TiKV/PD's starts from 9000.

Reply:
I've updated the title and the description.